### PR TITLE
chore: taxjar line item added

### PR DESCRIPTION
**Pull request by shantanu-kadam-epi**
Asana link:  https://app.asana.com/0/1199380658344177/1200577069882014

image

image

### CODE REVIEW

File: erpnext/accounts/doctype/sales_invoice_item/sales_invoice_item.json [hunk 1]
```diff
@@ -455,7 +455,6 @@
    "print_hide": 1
   },
   {
-   "default": ":Company",
```
Comment (sahil28297) [inline erpnext/accounts/doctype/sales_invoice_item/sales_invoice_item.json:458]: why was this removed?

**Review by sahil28297 — CHANGES_REQUESTED** [review]

**Comment by sahil28297** [comment]
@shantanu-kadam-epi fix sider issues

File: erpnext/accounts/doctype/sales_invoice_item/sales_invoice_item.json (hunk 1 above)
Comment (shantanu-kadam-epi) [reply erpnext/accounts/doctype/sales_invoice_item/sales_invoice_item.json:458]: In sales invoice, When I select another company than the default company & add the item in the item table,
because of this default company value for the cost center. It always returns the default company's cost center for the item.
& it throwing an error while submitting the sales invoice.

**Comment by neilLasrado** [comment]
@shantanu-kadam-epi I cant see the item name in the screenshot you attached, can you check that once?

File: erpnext/erpnext_integrations/taxjar_integration.py [hunk 2]
```diff
@@ -129,6 +137,12 @@ def set_sales_tax(doc, method):
 	sales_tax_exempted = hasattr(doc, "exempt_from_sales_tax") and doc.exempt_from_sales_tax \
 		or frappe.db.has_column("Customer", "exempt_from_sales_tax") and frappe.db.get_value("Customer", doc.customer, "exempt_from_sales_tax")
 
+	account_heads = get_account_heads(doc.company)
+	TAX_ACCOUNT_HEAD = account_heads['TAX_ACCOUNT_HEAD']
+
+	if not TAX_ACCOUNT_HEAD:
+		frappe.throw("Please add accounts in taxjar setting for company " + doc.company)
```
Comment (neilLasrado) [inline erpnext/erpnext_integrations/taxjar_integration.py]: throw(_("Please add accounts in taxjar setting for company " + doc.company))

**Review by neilLasrado — APPROVED** [review]

**Review by sahil28297 — APPROVED** [review]
